thavalon-webapp/src/styles/formStyles/InputElement.scss

```diff
@@ -0,0 +1,73 @@
+$input-border-valid-color: #4CAF50;
+$input-border-invalid-color: #960000;
+
+input[type="text"], input[type="email"], input[type="password"] {
+    -webkit-box-sizing: border-box;
+    -moz-box-sizing: border-box;
+    box-sizing: border-box;
+    width: 100%;
+    height: 1.75rem;
+    margin: 0 0 7px;
+    padding-left: 7px;
+    padding-top: 7px;
+    border: 1px solid #000;
+    border-radius: 10px;
+    background: #fff;
+    resize: none;
+    outline: none;
+    font-size: 1rem;
+    text-align: left;
+
+    &.content {
+        border-color: $input-border-invalid-color;
+
+        + label[placeholder]::before {
+            color: $input-border-invalid-color;
+        }
+    }
+
+    &:valid {
+        border-color: $input-border-valid-color;
+
+        + label[placeholder]::before {
+            color: $input-border-valid-color;
+        }
+    }
+
+    &:focus, &.content, &:valid {
+        + label[placeholder]::before {
+            -webkit-transition-duration: .2s;
+            transition-duration: .2s;
+            -webkit-transform: translate(0, -0.85rem) scale(0.9, 0.9);
+            -ms-transform: translate(0, -0.85rem) scale(0.9, 0.9);
+            transform: translate(0, -0.85rem) scale(0.9, 0.9);
+        }
+    }
+
+    + label[placeholder] {
+        display: block;
+        pointer-events: none;
+        line-height: 20px;
+        margin-top: -32px;
+        margin-bottom: 21px;
+        font-size: 1rem;
+
+        &::before {
+            content: attr(placeholder);
+            display: inline-block;
+            margin: 0 9px;
+            padding: 0 2px;
+            color: #333;
+            white-space: nowrap;
+            -webkit-transition: 0.3s ease-in-out;
+            transition: 0.3s ease-in-out;
+            background-image: -webkit-gradient((linear, left top, left bottom, from(#fff), to(#fff)));
+            background-image: -webkit-linear-gradient(top, #fff, #fff);
+            background-image: linear-gradient(to bottom, #fff, #fff);
```

Comment:
I don't know CSS enough to know, but can you have three separate `background-image` tags?

Reply:
I think this is how CSS handles support for multiple browsers

Reply:
Yeah, it'll try them in declaration order, so if the browser doesn't support `-webkit-gradient` it'll fall back to `-webkit-linear-gradient` and then `linear-gradient`. In this case, I think `-webkit-*` prefixes are used by Chrome and Safari, but not Firefox and so on